lupa/_lupa.pyx

```diff
@@ -1448,9 +1463,6 @@ cdef bint call_python(LuaRuntime runtime, lua_State *L, py_object* py_obj) excep
     cdef int i, nargs = lua.lua_gettop(L) - 1
     cdef tuple args
 
-    if not py_obj:
```

Comment:
Note that now all functions that call `call_python` already check if the object is missing.